### PR TITLE
update release information

### PR DESCRIPTION
**Pull request by blizzz**
- 21 saw its final release
- change to 4 weeks cycle

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]

File: admin_manual/release_schedule.rst [hunk 1]
```diff
@@ -26,14 +24,16 @@ Major releases are planned to be actively maintained for at least 8 months after
...
-Maintenance releases are scheduled in a 6 week cycle with one week before the release date having the freeze and RC 1.
+Maintenance releases are scheduled in a 4 week cycle with one week before the release date having the freeze and RC 1.
 
 Older versions
 --------------
 
 +----------+----------------+-------------+-------------------------+
 |          | release date   | end of life | latest version          |
 +==========+================+=============+=========================+
+| **21**   | 2021-02-22     | 2022-02     | 21.0.9 (2022-02-15)     |
```
Comment (nickvergessen) [inline admin_manual/release_schedule.rst:35]: I leave this to @AndyScherzinger

**Pull request merged**